### PR TITLE
[merc] support up to 64 effects

### PR DESCRIPTION
**Pull request by water111**

### CODE REVIEW

File: game/graphics/opengl_renderer/sprite/GlowRenderer.h [hunk 1]
```diff
@@ -76,7 +76,8 @@ class GlowRenderer {
 
   // max sprites should be 128 in simple sprite, plus 256 from aux = 384
   // 20 width = 20 * 20 = 400 sprites > 384.
-  // we multiply by 4 to get 16x as many max sprites (in-game the max is only 12x)
+  // we multiply by 2 to get 4x as many max sprites (in-game the max is 12x, but hopefully won't hit
+  // this limit)
```
Comment (ManDude) [inline game/graphics/opengl_renderer/sprite/GlowRenderer.h:80]: You didn't actually change it lol. The aux list is expanded [here](https://github.com/open-goal/jak-project/blob/master/goal_src/jak2/engine/gfx/sprite/sprite.gc#L149), I'm pretty sure you can just change the multiplier to reduce the sprite cap so you don't have to hope for anything.

**Review by ManDude — CHANGES_REQUESTED** [review]

File: game/graphics/opengl_renderer/sprite/GlowRenderer.h (hunk 1 above)
Comment (water111) [reply game/graphics/opengl_renderer/sprite/GlowRenderer.h]: Done. There is also the `simple-sprite-system` that can add glow sprites without using the aux list, but it looks like that is unchanged so 4x should be enough.

File: game/graphics/opengl_renderer/sprite/GlowRenderer.h [hunk 2]
```diff
@@ -76,7 +76,7 @@ class GlowRenderer {
 
   // max sprites should be 128 in simple sprite, plus 256 from aux = 384
   // 20 width = 20 * 20 = 400 sprites > 384.
-  // we multiply by 4 to get 16x as many max sprites (in-game the max is only 12x)
+  // we multiply by 2 to get 4x as many max sprites (in-game the max is 4x)
   static constexpr int kDownsampleBatchWidth = 20 * 4;
```
Comment (ManDude) [inline game/graphics/opengl_renderer/sprite/GlowRenderer.h:80]: You still forgot to actually change the number here. x3

Comment (water111) [reply game/graphics/opengl_renderer/sprite/GlowRenderer.h]: oof.

let's try a 3rd time...

**Review by ManDude — APPROVED** [review]

**Pull request merged**